python/py-allennlp/Portfile

```diff
@@ -0,0 +1,123 @@
+version             0.9.1
+revision            0
+name                py-${github.project}
+categories-append   textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         An open-source NLP research library, built on PyTorch.
+
+long_description    An Apache 2.0 NLP research library, built on\
+                    PyTorch, for developing state-of-the-art deep\
+                    learning models on a wide variety of linguistic\
+                    tasks.
+
+homepage            http://www.allennlp.org/
+
+checksums           rmd160  4fc2ffd4f892838d93b32d5a0a7931cb14149c82 \
+                    sha256  45e429692c5bad66f4ffa8dd4e204e0814df7ea48f50e02e91845daff89ba596 \
+                    size    7784697
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-boto3 \
+                    port:py${python.version}-conllu \
+                    port:py${python.version}-editdistance \
+                    port:py${python.version}-flaky \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-flask-cors \
+                    port:py${python.version}-ftfy \
+                    port:py${python.version}-gevent \
+                    port:py${python.version}-h5py \
+                    port:py${python.version}-jsonnet \
+                    port:py${python.version}-jsonpickle \
+                    port:py${python.version}-matplotlib \
+                    port:py${python.version}-nltk \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-numpydoc \
+                    port:py${python.version}-overrides \
+                    port:py${python.version}-parsimonious \
+                    port:py${python.version}-pytorch \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-responses \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-scikit-learn \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-spaCy \
+                    port:py${python.version}-sqlparse \
+                    port:py${python.version}-tensorboardX \
+                    port:py${python.version}-transformers \
+                    port:py${python.version}-tqdm \
+                    port:py${python.version}-tz \
+                    port:py${python.version}-unidecode \
+                    port:py${python.version}-word2number
+
+    depends_test-append \
+                    port:py${python.version}-flaky \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-responses
+
+    # Fix allennlp's dependecies on deprecated names pytorch-pretrained-bert|pytorch-transformers
```

Comment:
based on the discussion on this topic with upstream I don't feel confident that this is "production-ready"

Reply:
See above.